modules/OCPP/diagnostics_uploader.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Uploading"
 sleep 2
-curl --progress-bar -T "${3}" "${1}"
+curl --progress-bar --connect-timeout 20 -T "${3}" "${1}"
```

Comment:
you're using the const literal 20 in 3 different files, maybe this could be refactored to use one single setting, like a constants.sh or similar which gets included